storage-units/src/indexfile.rs

```diff
@@ -50,16 +50,28 @@ fn offset_hashes(bloom_size: u32) -> u64 {
 }
 
 // calculate the file offset from where the offsets are stored
-fn offset_offsets(bloom_size: u32, number_hashes: u32) -> u64 {
-    offset_hashes(bloom_size) + HASH_SIZE as u64 * number_hashes as u64
+fn offset_offsets(bloom_size: u32, number_hashes: u32, offset_type: &IndexOffsetType) -> u64 {
+    let standard = offset_hashes(bloom_size) + HASH_SIZE as u64 * number_hashes as u64;
+    match offset_type {
+        IndexOffsetType::Standard => standard,
+        IndexOffsetType::Unsorted => standard + OFF_SIZE as u64 * number_hashes as u64,
+    }
 }
 
 pub type IndexOffset = u32;
 
+#[derive(Clone, Debug)]
+pub enum IndexOffsetType {
+    Standard,
+    Unsorted,
+}
+
 // The parameters associated with the index file.
 // * the bloom filter size in bytes
+// * the ordinal index for the lookup (e.g. epoch)
 pub struct Params {
     pub bloom_size: u32,
+    pub ordinal: u32,
```

Comment:
1. Is it ok to name this parameter like this?
2. Is it ok to put it in lookup params?